cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -280,16 +285,30 @@ describe('Script Builder', () => {
           cy.log('select larger dataset')
           selectBucket('defbuck2')
           cy.getByTestID('flux-editor').contains(`from(bucket: "defbuck2")`, {
-            timeout: 30000,
+            timeout: 3000,
           })
 
-          runTest(3 * 500 + 2, 5 * 500, true)
+          runTest(3 * 500 + 2 + 60, 5 * 500 + 60, true)
```

Comment:
Because now have the original `writeData` of 60 (in the before_all of this test suite), plus the addition `writeData` to the same bucket for this specific test.